app/code/Magento/Sales/view/adminhtml/templates/order/address/form.phtml

```diff
@@ -15,7 +15,7 @@
         <span><?= $block->escapeHtml($block->getHeaderText()) ?></span>
     </legend>
     <br>
-    <div class="form-inline" data-mage-init='{"Magento_Sales/order/edit/address/form":{}}'>
```

Comment:
Why do you need to change this phtml file in the scope of your PR?
Is it correct or is it an unwanted git inclusion?